MobileWallet/Screens/Home/HomeViewController.swift

```diff
@@ -65,7 +65,7 @@ class HomeViewController: UIViewController, FloatingPanelControllerDelegate, Tra
 
     private let floatingPanelController = FloatingPanelController()
     private var grabberHandle: UIView!
-    private var selectedTransaction: Any?
+    private var selectedTransaction: TransactionProtocol?
```

Comment:
I think you could name the protocol just `Transaction` (if it's not in use of course).